File: SDKs/amazonaws.com/rds/2014-10-31/typescript/src/sdk/models/shared/dbclusterbacktrackmessage.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { DbClusterBacktrack } from "./dbclusterbacktrack";



// DbClusterBacktrackMessage
/** 
 * Contains the result of a successful invocation of the <code>DescribeDBClusterBacktracks</code> action.
**/
export class DbClusterBacktrackMessage extends SpeakeasyBase {
  @SpeakeasyMetadata({ elemType: DbClusterBacktrack })
  dbClusterBacktracks?: DbClusterBacktrack[];

  @SpeakeasyMetadata()
  marker?: string;
}

File: SDKs/amazonaws.com/rds/2014-10-31/python/src/sdk/models/shared/dbclusterbacktrackmessage.py
from dataclasses import dataclass, field
from typing import List,Optional
from . import *


@dataclass
class DbClusterBacktrackMessage:
    r"""DbClusterBacktrackMessage
    Contains the result of a successful invocation of the <code>DescribeDBClusterBacktracks</code> action.
    """
    
    db_cluster_backtracks: Optional[List[DbClusterBacktrack]] = field(default=None)
    marker: Optional[str] = field(default=None)
    
